runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
+* Moving processes to respective cgroups.
```

Comment:
On Wed, Dec 16, 2015 at 11:06:00AM -0800, Jonathan Boulle wrote:
> > +* Moving processes to respective cgroups.
> 
> This is also affected by `cgroupsPath`, right?

I don't think so (see my breakdown here [1])

> Actually I find this list pretty unclear as-is…

And so has everyone else that has reviewed it ;).  So I think we need
to either drop it completely or replace it with something like the
shell example in [2].

[1]: https://github.com/opencontainers/specs/pull/247#discussion_r47373091
[2]: https://github.com/opencontainers/specs/pull/247#discussion_r47373091